src/gradient.h

```diff
@@ -10,6 +10,62 @@
 #ifndef GRADIENT_H
 #define GRADIENT_H
 
+/** Enumeration of Gradient type used for determine gradient function to use.
+ */
+typedef unsigned char GradientType;
+#define GRADIENT_TITLE_ACTIVE   1
+#define GRADIENT_TITLE          2
+#define GRADIENT_COUNT          3
```

Comment:
Why not start at 0?

Reply:
Ok my bad, initially I added GRADIENT_DUMMY for component where I don't test the new gradient draw mode.